### ISSUE TITLE
Add Pinch to Zoom feature

### ISSUE THREAD

**Issue by nerdic-coder** [TeamHive/stencil-pdf-viewer#17]
Would be nice if Multi-touch devices could do Pinch to Zoom instead of using zoom buttons.

**Comment by kustrun** [comment]
I agree. Tried today to implement this features with [Hammer.JS](https://hammerjs.github.io/) like @justintrue started on [feat/pinch-zoom](/TeamHive/pdf-viewer/tree/feat/pinch-zoom) but I found out the code in **pdf-viewer-assets** is not commited...
